net-misc/asterisk-opus/asterisk-opus-13.0.1.3.0.ebuild

```diff
@@ -0,0 +1,52 @@
+# Copyright 1999-2020 Gentoo Authors
+# Distributed under the terms of the GNU General Public License v2
+
+EAPI="7"
+
+inherit multilib
```

Comment:
One more thing, **sigh** I need to focus more when viewing these.

What's the benefit of having multilib here, especially if this is a binary package? It doesn't seem to be used in this ebuild anyway.

Reply:
Seriously my bad.